app/models/vote.rb

```diff
@@ -0,0 +1,6 @@
+class Vote < ApplicationRecord
+  belongs_to :user
+  belongs_to :votable, polymorphic: true
+
+  validates :value, inclusion: {in: [-1, 0, 1]}
```

Comment:
A user shouldn't be able to vote on the same votable more than once, so I would add a validation checking the uniqueness of those two fields to this model.